lib/elixir/lib/option_parser.ex

```diff
@@ -53,6 +53,7 @@ defmodule OptionParser do
 
     * `:switches` or `:strict` - see the "Switch definitions" section below
     * `:aliases` - see the "Aliases" section below
+    * `:allow_nonexistent_atoms` - see "Parsing undefined switches" section below
```

Comment:
"see "Parsing undefined switches" ..." -> "see **the** "Parsing undefined switches" ...", wdyt?